composer.json

```diff
@@ -28,7 +29,7 @@
         "codeception/mockery-module": "^0.4.0",
         "dms/phpunit-arraysubset-asserts": "^0.1.0",
         "friendsofphp/php-cs-fixer": "^2.16",
-        "orchestra/testbench": "^4.0 || ^5.0 || ^6.0",
+        "orchestra/testbench": "^3.5",
```

Comment:
It looks like this change should not be here. Also tests are not running, did you merge changes from master?